ios/TestApp/TestApp/Benchmark.mm

```diff
@@ -61,16 +69,14 @@ + (void)benchmarkWithModel:(NSString*)modelPath {
   if (FLAGS_print_output) {
     std::cout << module.forward(inputs) << std::endl;
```

Comment:
change this to UI_LOG as well? or just remove this? how do we support gflags in iOS test app?

Reply:
Right now, all the gflag values are hardcoded. Thinking about using a JSON file to make it configurable.